src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidatorTest.java

```diff
@@ -20,6 +20,14 @@ public void should_return_warning_when_email_is_not_provided() {
         });
     }
 
+    @Test
+    public void should_return_warning_when_email_is_invalid() {
+        assertSoftly(softly -> {
```

Comment:
soft assertion not needed as there is only thing to assert ?

Reply:
Have fixed